IBF-Typhoon-model/src/typhoonmodel/pipeline.py

```diff
@@ -56,14 +56,18 @@
 @click.option('--path', default='./', help='main directory')
 @click.option('--remote_directory', default=None, help='remote directory for ECMWF forecast data') #'20210421120000'
 @click.option('--typhoonname', default='SURIGAE',help='name for active typhoon')
-@click.option('--LatMax', default=19,help='maximum extent latitude')
-@click.option('--LatMin', default=6,help='minimum extent latitude')
-@click.option('--LonMax', default=127,help='maximum extent longtiude')
-@click.option('--LonMin', default=118,help='minimum extent longtiude')
+@click.option('--debug_flag', default=False,help='setting for DEBUG option')
```

Comment:
At the moment, you have to pass `--debug_flag True` in order to trigger debugging. If you omit `default=False` then the parameter will become an actual flag where you only have to pass `--debug_flag` (without the `True`) to turn on debugging. 

Also, usually flags omit "flag" from the name, i.e. I would just call it `debug` and not `debug_flag`.